test_conformance/conversions/test_conversions.c

```diff
@@ -45,6 +45,7 @@
 #include <sys/mman.h>
 #endif
 #include <time.h>
+#include <vector>
```

Comment:
This file has `.c` extensions, but now it includes C++ STL header.
Could you please note this file in #25 or may be it is better to change it to `.cpp` just now?